src/main/java/duke/storage/Storage.java

```diff
@@ -32,10 +32,12 @@ public List<String> readFile() {
         try {
             f.createNewFile();
             Scanner s = new Scanner(f);
```

Comment:
Generally good code quality, but maybe can consider using File file, Scanner scanner and even below Filewriter filewriter to make it even easier for others to follow